core/src/main/java/nu/yona/server/rest/TestController.java

```diff
@@ -101,7 +102,7 @@ private EmailResourceAssembler createEMailResourceAssembler()
 		{
 			return new ResponseEntity<>(HttpStatus.NOT_FOUND);
 		}
-		return createOkResponse(FirebaseMessageDto.createInstance(lastMessage.get()), createFirebaseMessageResourceAssembler());
+		return createOkResponse(FirebaseMessageDto.createInstance(lastMessage.get()), createFirebaseMessageRepresentationModelAssembler());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "lastMessage.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS3655)